dspace_rest_client/client.py

```diff
@@ -1163,3 +1183,151 @@ def update_resource_policy_group(self, policy_id, group_uuid):
         body = f'{self.API_ENDPOINT}/eperson/groups/{group_uuid}'
         r = self.api_put_uri(url, None, body, False)
         return r
+
+    def get_clarinlruallowances(self):
+        """
+        Fetch all clarinlruallowances.
+        """
+        url = f'{self.API_ENDPOINT}/core/clarinlruallowances'
+        try:
+            response = self.api_get(url)
+            data = parse_json(response)
+            allowances = data.get('_embedded', {}).get('clarinlruallowances')
+            if allowances:
+                logging.info(f"Fetched {len(allowances)} CLARIN LRU allowances.")
+                return allowances
+            logging.warning("No CLARIN LRU allowances found.")
+        except Exception as e:
+            logging.error(f"Error fetching CLARIN LRU allowances [{url}]: {e}")
+        return None
+
+    def get_clarinlruallowances_by_bitstream_and_user(self, bitstream_uuid, user_uuid):
+        """
+        Fetch user allowances for a specific bitstream and user.
+        """
+        url = f'{self.API_ENDPOINT}/core/clarinlruallowance/search/byBitstreamAndUser'
+        params = {'bitstreamUUID': bitstream_uuid, 'userUUID': user_uuid}
+        try:
+            response = self.api_get(url, params=params)
+            data = parse_json(response)
+            allowances = data.get('_embedded', {}).get('clarinlruallowances')
+            if allowances:
+                logging.info(f"Found {len(allowances)} user allowance(s).")
+                return allowances
+            logging.warning(f"No user allowances found for user: {user_uuid} and bitstream: {bitstream_uuid}")
```

Comment:
The `_logger` is used in the methods above, why are you using `logging`? Be consistent

Reply:
Actually, in that class, originally logging was using. But oneday jm added there _logger. So ther eis mishmash in logging now. By I used in mu class _logger, because of the jm...